composer.json

```diff
@@ -109,6 +109,9 @@
             "drupal/redirect": {
                 "Redirection issue when interface language is different from content language": "https://www.drupal.org/files/issues/2020-06-01/redirect-interface_language_different_from_content_language_2991423-13.patch"
             },
+            "drupal/socialbase": {
+                "#3259374 - add temp patch for symfony mailer theme changes until it's merged together": "https://git.drupalcode.org/project/socialbase/-/merge_requests/88.diff"
```

Comment:
probably best to get this in a release (or use a non rolling patch)

Reply:
This one is now in 2.5.10